lib/item.rb

```diff
@@ -1,5 +1,3 @@
-# frozen_string_literal: true
```

Comment:
I like removing these, they don't seem to impact our functionality at all.

Reply:
Agreed ^^